functions/routes/ecom/modules/create-transaction.js

```diff
@@ -108,7 +108,7 @@ exports.post = ({ appSdk }, req, res) => {
     country: to.country_code ? to.country_code.toLowerCase() : 'br',
     zipcode: to.zip,
     street_number: String(to.number) || 's/n',
-    complementary: to.complement
+    complementary: to.complement ? to.complement : '0' 
```

Comment:
```suggestion
    complementary: to.complement ? to.complement : null
```

Não aceitam `''` mas pelos exemplos da doc aceitam `null`:
https://docs.pagar.me/reference#criar-transacao
Acho que é o mais correto nesse caso.